src/main/java/org/folio/inventory/MarcBibUpdateConsumerVerticle.java

```diff
@@ -0,0 +1,96 @@
+import static org.folio.inventory.dataimport.util.KafkaConfigConstants.OKAPI_URL;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Promise;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.inventory.dataimport.cache.MappingMetadataCache;
+import org.folio.inventory.dataimport.consumers.MarcBibUpdateKafkaHandler;
+import org.folio.inventory.dataimport.handlers.actions.InstanceUpdateDelegate;
+import org.folio.inventory.storage.Storage;
+import org.folio.kafka.GlobalLoadSensor;
+import org.folio.kafka.KafkaConfig;
+import org.folio.kafka.KafkaConsumerWrapper;
+import org.folio.kafka.KafkaTopicNameHelper;
+import org.folio.kafka.SubscriptionDefinition;
+
+public class MarcBibUpdateConsumerVerticle extends AbstractVerticle {
+  private static final Logger LOGGER = LogManager.getLogger(MarcBibUpdateConsumerVerticle.class);
+  private static final GlobalLoadSensor GLOBAL_LOAD_SENSOR = new GlobalLoadSensor();
+  private static final String SRS_MARC_BIB_TOPIC_NAME = "srs.marc-bib";
+  private final int loadLimit = getLoadLimit();
+  private KafkaConsumerWrapper<String, String> marcBibUpdateConsumerWrapper;
+
+  @Override
+  public void start(Promise<Void> startPromise) {
+    JsonObject config = vertx.getOrCreateContext().config();
+    KafkaConfig kafkaConfig = getKafkaConfig(config);
+    marcBibUpdateConsumerWrapper = createConsumer(kafkaConfig, SRS_MARC_BIB_TOPIC_NAME);
+    HttpClient client = vertx.createHttpClient();
+    Storage storage = Storage.basedUpon(config, client);
+    InstanceUpdateDelegate instanceUpdateDelegate = new InstanceUpdateDelegate(storage);
+    String mappingMetadataExpirationTime = getCacheEnvVariable(config,"inventory.mapping-metadata-cache.expiration.time.seconds");
+    MappingMetadataCache mappingMetadataCache = new MappingMetadataCache(vertx, client, Long.parseLong(mappingMetadataExpirationTime));
+
+    MarcBibUpdateKafkaHandler marcBibUpdateKafkaHandler = new MarcBibUpdateKafkaHandler(instanceUpdateDelegate, mappingMetadataCache);
+
+    marcBibUpdateConsumerWrapper.start(marcBibUpdateKafkaHandler, constructModuleName())
+      .onFailure(startPromise::fail)
+      .onSuccess(ar -> startPromise.complete());
+  }
+
+  private KafkaConsumerWrapper<String, String> createConsumer(KafkaConfig kafkaConfig, String srsMarcBibTopicName) {
+    SubscriptionDefinition subscriptionDefinition = KafkaTopicNameHelper.createSubscriptionDefinition(
+        kafkaConfig.getEnvId(), KafkaTopicNameHelper.getDefaultNameSpace(), srsMarcBibTopicName);
+
+    return KafkaConsumerWrapper.<String, String>builder()
+      .context(context)
+      .vertx(vertx)
+      .kafkaConfig(kafkaConfig)
+      .loadLimit(loadLimit)
+      .globalLoadSensor(GLOBAL_LOAD_SENSOR)
+      .subscriptionDefinition(subscriptionDefinition)
+      .build();
+  }
```

Comment:
The full topic name will have the format `<env>.<tenant>.srs.marc-bib`.
But in this method, you defined it as `<env>.<namespace>.<tenant>.srs.marc-bib`

Reply:
done